scripts/services/getConfig.ts

```diff
@@ -253,6 +253,30 @@ export const getConfig: T.ExpectedExports.getConfig = compat.getConfig({
       }
     }
   },
+  "watchtowers": {
```

Comment:
Wait, I thought we were doing this in actions? What's this config section for?

Reply:
you have both options